src/shared/Dropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import { createSignal, Show, Component, JSX } from 'solid-js';
+interface Props {
+    options: string[];
+    optionClickHandlers: (() => void)[];
+    buttonContent: JSX.Element;
+  }
+const [open, setOpen] = createSignal(false);
+
+const handleClick = () => {
+    setOpen(!open())
+  }
+const Dropdown: Component<Props> = (props) => (
+  <div class="relative">
+      <button onClick={handleClick}>{props.buttonContent}</button>
+      <Show when={open()}>
+        <ul class="bg-black border border-gray-400 rounded-lg py-2 absolute mt-2 z-10">
+          {props.options.map((option, index) => (
+            <li
```

Comment:
Is keyboard navigation/accessibility something this project is concerned with? If so, how does using clickable `<li />`s work with keyboard navigation?